packages/react-query/src/index.ts

```diff
@@ -1 +1,4 @@
+const suspensiveReactQueryVersion = '4'
+
 export * from '@suspensive/react-query-4'
+export { suspensiveReactQueryVersion }
```

Comment:
> This was added to check the version of @suspensive/react-query

In my opinion, version of @suspensive/react-query should be '2.2.3' not '4'. In my opinion, name of this variable should be `targetTanStackReactQueryVersion` "if need".

but I have 2 ideas to remove that variable.

1. if we can read file text string and parse `export * from '@suspensive/react-query-4'` or `export * from '@suspensive/react-query-5'`,
2. if we use `import packageJson from "@suspensive/react-query-4/package.json"` or `import packageJson from "@suspensive/react-query-5/package.json"`
    ```ts
    import packageJson from "@suspensive/react-query-4/package.json"
    packageJson.name // This will be "@suspensive/react-query-4"
    ```

 we won't need to use that variable. Could you try that please?

```suggestion
export * from '@suspensive/react-query-4'
```

Reply:
The version here refers to the currently set (switched) version of @suspensive/react-query (the version of dist/index.js). It determines compatibility by comparing the currently set version (dist/index.js) with the version of @tanstack/react-query installed in the environment.

1. `export * from '@suspensive/react-query-4'` or `export * from '@suspensive/react-query-5'` does not include `package.json`.
2. Accessing `import packageJson from "@suspensive/react-query-4/package.json"` cannot retrieve the currently set version.

For these two reasons, it is not possible to retrieve the currently set (switched) version. Thus, I had to use variables, and despite attempts to retrieve it without variable settings, it was not possible.

- It is not possible to dynamically access the package.json of the package set in dist/index.js.

How about using the following method?

```ts
import packageJson from '@suspensive/react-query-4/package.json'
export * from '@suspensive/react-query-4'
export { packageJson }
```